src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -125,8 +125,8 @@ protected function tokensMatch($request)
      * Add the CSRF token to the response cookies.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  \Illuminate\Http\Response  $response
-     * @return \Illuminate\Http\Response
+     * @param  \Symfony\Component\HttpFoundation\Response  $response
```

Comment:
No other methods hint `Response`, but only `Request`.
The reason is that the method itself actually does not require any Illuminate-specific methods of Response